src/hotspot/share/opto/divnode.cpp

```diff
@@ -1626,20 +1630,6 @@ Node* ModFloatingNode::replace_with_con(PhaseIterGVN* phase, const Type* con) {
   CallProjections projs;
   extract_projections(&projs, false, false);
   phase->replace_node(projs.fallthrough_proj, in(TypeFunc::Control));
-  if (projs.fallthrough_catchproj != nullptr) {
-    phase->replace_node(projs.fallthrough_catchproj, in(TypeFunc::Control));
-  }
-  if (projs.fallthrough_memproj != nullptr) {
-    phase->replace_node(projs.fallthrough_memproj, in(TypeFunc::Memory));
-  }
-  if (projs.catchall_memproj != nullptr) {
-    phase->replace_node(projs.catchall_memproj, C->top());
-  }
-  if (projs.fallthrough_ioproj != nullptr) {
-    phase->replace_node(projs.fallthrough_ioproj, in(TypeFunc::I_O));
-  }
-  assert(projs.catchall_ioproj == nullptr, "no exceptions from floating mod");
-  assert(projs.catchall_catchproj == nullptr, "no exceptions from floating mod");
```

Comment:
Why were you able to remove this?

Reply:
Great! When I remove code like that, then I often just leave a "info" code comment, just to help the reviewers - you don't need to do that, it's just an idea ;)